src/generator.rs

```diff
@@ -285,9 +285,14 @@ After=systemd-zram-setup@{zram_device}.service
 [Mount]
 What=/dev/{zram_device}
 Where={mount_point:?}
+Options={mount_options:?}
 ",
             zram_device = device.name,
             mount_point = device.mount_point.as_ref().unwrap(),
+            mount_options = match &device.mount_options {
```

Comment:
`unwrap_or()`, or, better yet, omit `Options=` if `mount_options` is `None`

Reply:
> omit Options= if mount_options is None

May I have some sample code snippet?